src/data_helper.py

```diff
@@ -105,7 +104,14 @@ def get_train_generator(self, batch_size):
                     # Augment the image `img` here
 
                     # Convert to RGB and normalize
-                    img_array = np.asarray(img.convert("RGB"), dtype=np.float32) / 255
+                    img_array = np.asarray(img.convert("RGB"), dtype=np.float32)
+
+                    img_array = img_array[:, :, ::-1]
+                    # Zero-center by mean pixel
+                    img_array[:, :, 0] -= 103.939
+                    img_array[:, :, 1] -= 116.779
+                    img_array[:, :, 2] -= 123.68
```

Comment:
Nice!! :D But I think didn't get the whole story around why some ppl decides to normalize the values, other decides to remove the mean other do something else blah blah. I'm still a bit confused even if I understand that here we are forced to do that for finetunning.